packages/core/src/test/shared/utilities/pathFind.test.ts

```diff
@@ -42,4 +41,90 @@ describe('pathFind', function () {
         const regex = /bin[\\\/](code|code-insiders)$/
         assert.ok(regex.test(vscPath), `expected regex ${regex} to match: "${vscPath}"`)
     })
+
+    describe('findSshPath', function () {
+        let previousPath: string
+
+        beforeEach(function () {
+            previousPath = testutil.copyEnv().PATH!
```

Comment:
if we aren't going to store the whole env, we can just save PATH directly.
```suggestion
            previousPath = testutil.env.PATH
```